packages/auth/src/graphql/plugin/index.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  ApolloServerPlugin,
+  GraphQLRequestContextDidEncounterErrors,
+  GraphQLRequestContextDidResolveOperation,
+  GraphQLRequestListener
+} from '@apollo/server'
+import { Logger } from 'pino'
+import { v4 as uuid } from 'uuid'
+
+import { GraphQLErrorCode } from '../errors'
+
+export class LoggingPlugin implements ApolloServerPlugin {
+  private logger
+
+  constructor(logger: Logger) {
+    this.logger = logger
+  }
+
+  async requestDidStart(): Promise<GraphQLRequestListener<never>> {
+    const requestId = uuid()
+    const logger = this.logger
+    let operation: string | null
+
+    return {
+      async didResolveOperation(
+        context: GraphQLRequestContextDidResolveOperation<never>
+      ) {
+        operation = context.operationName
+        logger.info({
```

Comment:
should this be a debug log, and should it include the input variables?
CC @sabineschaller (since its related to your change for `backend` as well)

Reply:
I would expect this to stay info personally, especially since the `requestId` is referenced in the `didEncounterErrors` logs. Helps tell the whole story I think.

Reply:
I looked at the [diagram Blair shared with us](https://i.stack.imgur.com/z5Fim.png) and I agree. That should be `info`.